docs/user/alerting/create-and-manage-rules.asciidoc

```diff
@@ -150,6 +149,33 @@ Some cases exist where the variable values will be "escaped", when used in a con
 
 Mustache also supports "triple braces" of the form `{{{variable name}}}`, which indicates no escaping should be done at all.  Care should be used when using this form, as it could end up rendering the variable content in such a way as to make the resulting parameter invalid or formatted incorrectly.
 
+[float]
```

Comment:
Seems like this could be organized better

- there's a screenshot and narrative "Some cases exist ..." between the two sets of action variables; seems like that should not be BETWEEN them, but probably before or after them?
- it's not clear why there are two separate lists, so I think we'll need a short explanation that the top is for alerts that are **NOT** summaries, and the bottom **IS** for summaries - you get one set or the other, depending on whether the alerts are summarized.

Reply:
done